config.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   redirectUrl: {
-    development: "https://6e0b93f6.ngrok.io/api/getAccessToken",
+    development: "https://8b0a5404.ngrok.io/api/getAccessToken",
```

Comment:
Misschien is het handig om een manier te vinden om deze in de gitignore te krijgen, aangezien hij anders per PR verandert. Misschien de redirectUrl splitsen van de rest van de config file o.i.d.?